### ISSUE TITLE
Proxy Support

### ISSUE THREAD

**Issue by VatsalJagani** [CrossRealms/Splunk-App-Auto-Update-MaxMind-Database#12]
To enable proxy support currently, users can manually make the following changes.

bin/mmdb_utils.py - line no. 171
```
       proxies = None
        '''
        proxies = {
            "http" : "<proxy-supported-schema http|https>://<username>:<password>@<ip-address>:<port>",
            "https" : "<proxy-supported-schema http|https>://<username>:<password>@<ip-address>:<port>"
        }
        '''
```

Comment the first line and uncomment below lines like this
http proxy example:
```
        # proxies = None
        proxies = {
            "http" : "http://10.0.0.1:3128",
            "https" : "http://10.0.0.1:3128"
        }
```

https proxy example:
```
        # proxies = None
        proxies = {
            "http" : "https://10.0.0.1:3128",
            "https" : "https://10.0.0.1:3128"
        }
```

### FAQ

* *Getting SSLError while using proxy*

Make sure you proxy settings is correct and it is not using both http and https scheme at the same time.
Below wrong proxy configuration may result the SSLError
```
        # proxies = None
        proxies = {
            "http" : "http://10.0.0.1:3128",
            "https" : "https://10.0.0.1:3128"
        }
```

**Comment by VatsalJagani** [comment]
In some case, proxy setting without schema seems to work as mentioned in https://github.com/CrossRealms/Splunk-App-Auto-Update-MaxMind-Database/issues/25 
like

```
proxies = {
    'https' : '192.168.25.5:8080'
}
```

**Comment by VatsalJagani** [comment]
We have just released the new version of v4.0.0 on Splunkbase which has officially added Proxy support to be configured from UI.

https://splunkbase.splunk.com/app/5482

See the release notes and other documentation here - https://github.com/CrossRealms/Splunk-App-Auto-Update-MaxMind-Database
